components/speedreader/resources/panel/components/tts-control/index.tsx

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+import * as S from './style'
+
+import getToolbarAPI, { TtsSettings, PlaybackSpeed, PlaybackState } from '../../api/browser'
+import { Playback, PlaybackControl, PlaybackSpeedControl } from '../lists'
+import { getLocale } from '$web-common/locale'
+
+interface TtsControlProps {
+  ttsSettings: TtsSettings
+  onTtsVoiceChange: (voice: string) => void;
+  onTtsSpeedChange: (speed: PlaybackSpeed) => void
+  onClose: () => void
+}
+
+function TtsControl(props: TtsControlProps) {
+  const [voices, setVoices] = React.useState(speechSynthesis.getVoices())
+  speechSynthesis.onvoiceschanged = () => {
+    setVoices(speechSynthesis.getVoices())
+  }
+
+  const [playbackState, setPlaybackState] = React.useState<PlaybackState>(PlaybackState.kStopped)
+  getToolbarAPI().dataHandler.getPlaybackState().then(res => setPlaybackState(res.playbackState))
+  getToolbarAPI().eventsRouter.setPlaybackState.addListener((state: PlaybackState) => {
+    setPlaybackState(state)
+  })
+
+  return (
+    <S.Box>
+      <PlaybackControl
+        playbackState={playbackState}
+        onClick={(command: Playback) => {
+          switch (command) {
+            case Playback.Play:
+              getToolbarAPI().dataHandler.play()
+              break
+            case Playback.Pause:
+              getToolbarAPI().dataHandler.pause()
+              break
+            case Playback.Stop:
+              getToolbarAPI().dataHandler.stop()
+              break
+            case Playback.Forward:
+              getToolbarAPI().dataHandler.forward()
+              break
+            case Playback.Rewind:
+              getToolbarAPI().dataHandler.rewind()
+              break
+          }
+        }}
+      />
+      <S.VDelemiter />
```

Comment:
Same feedback on this element like the other one. Let's simplify this by adding styles to the parent or a wrapper div.